apps/checkout/src/components/RadioGroup.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Radio, RadioOption, RadioProps } from "./Radio";
+
+interface RadioGroupProps
+  extends Pick<RadioProps, "onSelect" | "selectedValue" | "alignment"> {
+  options: RadioOption[];
+  label: string;
+}
+
+export const RadioGroup: React.FC<RadioGroupProps> = ({
```

Comment:
I know this is temporary but let's use compound components pattern for this one in the ui kit, cc @adrian-potepa

Reply:
Okay